docs/api/model.md

```diff
@@ -268,6 +268,7 @@ Run an aggregation method on the specified field
 | [options.distinct] | boolean | Applies DISTINCT to the field being aggregated over |
 | [options.transaction] | Transaction |  |
 
+__Returns:__ Returns the result of the aggregate method cast to `options.dataType`, unless `options.plain` is set to false, in which case the raw data result is returned.
```

Comment:
You should not modify `docs/api/*` directly, it's generated from source comments.